src/documents/DeprecatedIn3_0.md

```diff
@@ -41,6 +41,24 @@ The following files and all their contents are deprecated:
 * src/framework/renderer/js/fluidRenderer.js
 * src/framework/renderer/js/RendererUtilities.js
 
+The following bundles are no longer produced as part of the build process:
+
+* dist/infusion-all-no-jquery.min.js
+* dist/infusion-all-no-jquery.min.js.map
+* dist/infusion-all.min.js
+* dist/infusion-all.min.js.map
+* dist/infusion-framework-no-jquery.min.js
+* dist/infusion-framework-no-jquery.min.js.map
+* dist/infusion-framework.min.js
+* dist/infusion-framework.min.js.map
+* dist/infusion-uio-no-jquery.min.js
+* dist/infusion-uio-no-jquery.min.js.map
+* dist/infusion-uio.min.js
+* dist/infusion-uio.min.js.map
+
+We only produce minified javascript bundles now, so if you previously used one of the above minified files, you
+can simply update the relative path/filename to remove the `min`, i.e. `dist/infusion-all.js`.
```

Comment:
The section that these are in, is actually a sub section of Renderer. This should probably be in a new Bundles or Builds section that would be a sibling to Pager, Renderer and Fast XML Pull.

Reply:
Good catch, I moved it to its own section.  It's now similar to the pager section, in that it has a "bundles" heading and then details about "files".